cloud-sql/mysql/client-side-encryption/snippets/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
Please remove the header from __init__.py - [it's allowed to be empty](https://github.com/GoogleCloudPlatform/python-docs-samples/blob/8b72c12d77fe68592552a36f673cb0d9d9806874/.github/header-checker-lint.yml#L3) (and if the bot is angry, lmk because that means I got the config wrong)

Reply:
Why don't we want headers in `__init__` files?